VideoLocker/res/values/strings.xml

```diff
@@ -428,7 +428,7 @@
     <!-- Accessibility Label for Pause button of Video Player -->
     <string name="video_player_pause">Pause</string>
     <!-- Accessibility Label for Rewind button on Video Player bottom bar -->
-    <string name="video_player_rewind">Rewind</string>
+    <string name="video_player_rewind">Rewind 30 seconds</string>
```

Comment:
@catong I think the intention here was to be a more general rewind button but currently it's hardcoded at 30 seconds. I don't really see any reason to change the 30s seek in the near future.
<img width="263" alt="screen shot 2016-08-12 at 7 35 31 am" src="https://cloud.githubusercontent.com/assets/7101723/17621147/7629264c-605f-11e6-808b-f5269d8b50a0.png">